enterprise/server/test/integration/ci_runner/ci_runner_test.go

```diff
@@ -770,8 +770,8 @@ func TestRunAction_TargetRepoOnly(t *testing.T) {
 		"--workflow_id=test-workflow",
 		"--action_name=Print args",
 		"--trigger_event=push",
-		"--target_repo_url=file://" + repoPath,
-		"--target_branch=master",
+		"--pushed_repo_url=file://" + repoPath,
+		"--pushed_branch=master",
```

Comment:
(following the previous comment)

```suggestion
		"--target_repo_url=file://" + repoPath,
		"--target_branch=master",
		"--pushed_repo_url=file://" + repoPath,
		"--pushed_branch=master",
```